tests/onnx/quantization/common.py

```diff
@@ -0,0 +1,84 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import List
+
+import os
+import tempfile
+
+import numpy as np
+import onnx
+import onnxruntime as rt
+
+from tests.common.helpers import TEST_ROOT
+
+from tests.onnx.test_nncf_graph_builder import check_nx_graph
+
+from nncf.experimental.post_training.api.dataloader import DataLoader
+from nncf.experimental.post_training.compression_builder import CompressionBuilder
+from nncf.experimental.onnx.algorithms.quantization.min_max_quantization import ONNXMinMaxQuantization
+from nncf.experimental.post_training.algorithms.quantization import MinMaxQuantizationParameters
+from nncf.experimental.post_training.algorithms.quantization import PostTrainingQuantization
+from nncf.experimental.post_training.algorithms.quantization import PostTrainingQuantizationParameters
+from nncf.experimental.onnx.graph.nncf_graph_builder import GraphConverter
+
+REFERENCE_GRAPHS_TEST_ROOT = 'data/reference_graphs/quantization'
+
+
+class TestDataloader(DataLoader):
```

Comment:
Functions and classes that start with `test*` or `Test*` become treated by pytest as test cases. Rename to `DataloaderForTest`.

Reply:
Done